### PR TITLE
LPS-133447 | master

### PR DESCRIPTION
**Pull request by ThalitaCeli**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase [hunk 1]
```diff
@@ -1358,6 +1358,34 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-132425. Web Content folder search results show folder title in main"
+	@priority = "4"
+	test ViewWebContentFolderTitle {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase:1363]: Should be SearchWebContentFolder

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase [hunk 2]
```diff
@@ -1358,6 +1358,34 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-132425. Web Content folder search results show folder title in main"
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase:1361]: Should be This is a test for LPS-132425. Search web content folder on page

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase [hunk 3]
```diff
@@ -1358,6 +1358,34 @@ definition {
...
+		task ("Add a Web Content Folder") {
+			ProductMenu.gotoPortlet(
+				category = "Content &amp; Data",
+				panel = "Site Administration",
+				portlet = "Web Content");
+
+			WebContentFolder.addCP(
+				folderDescription = "WC Folder Description",
+				folderName = "WC Folder Name");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase:1375]: Use json macro to create a web content folder

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase [hunk 4]
```diff
@@ -1358,6 +1358,34 @@ definition {
...
+
+		task ("Search Web Content Folder in main") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase:1377]: Search the web content folder on page

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase [hunk 5]
```diff
@@ -1358,6 +1358,34 @@ definition {
...
+			Navigator.openURL();
+
+			SearchPortlets.searchEmbedded(searchTerm = "WC Folder Name");
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase:1381]: Add task message **View the web content folder shown in search results**

**Comment by Tim-Cao** [comment]
Hi @ThalitaCeli, Could you move the case to WebContentSearch testcase file?

**Comment by david-gutierrez-mesa** [comment]
Hi @ThalitaCeli PR should start by LPS number. Example: "LPS-133447 | master" and not "Automated Test - LPS-133447 | master"

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentsearch/WebContentSearch.testcase [hunk 6]
```diff
@@ -250,6 +250,34 @@ definition {
...
+	test SearchWebContentFolder {
+		task ("Add a Web Content Folder") {
+			JSONWebcontent.addFolder(
+				folderName = "WC Folder Name",
+				groupName = "Guest");
+
+			JSONWebcontent.addWebContent(
+				content = "WC WebContent Content",
+				groupName = "Guest",
+				title = "WC WebContent Title");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentsearch/WebContentSearch.testcase:264]: Unneeded step

**Comment by Tim-Cao** [comment]
See changes https://github.com/liferay-echo/liferay-portal/compare/e08ba4a175335e8df194117501a62359c1adaa4e...fa742cd59f307203181069670d25d963c3540dc5
https://github.com/liferay-echo/liferay-portal/pull/4922

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**